src/bootstrap/plugin.loader.ts

```diff
@@ -17,6 +17,7 @@ import { FetchClassChannelsPlugin } from '../app/plugins/fetchclasschans.plugin'
 import { AddClassChannelsPlugin } from '../app/plugins/addclasschans.plugin';
```

Comment:
should be in its own pr

Reply:
Yeah what the heck?  That is a completely different branch.